trunk/SUAVE/Methods/Flight_Dynamics/Static_Stability/Approximations/Tube_Wing/taw_cmalpha.py

```diff
@@ -167,16 +167,16 @@ def taw_cmalpha(geometry,mach,conditions,configuration):
     vehicle = SUAVE.Vehicle()
     wing = SUAVE.Components.Wings.Wing()
     wing.tag = 'main_wing'
-    wing.areas.reference           = 5500.0 * Units.feet**2
```

Comment:
These syntax changes are wrong. But this module test should probably just be removed.

Reply:
I am going to revert...but we should open a PR to fix this